packages/core/src/App/AppContent.tsx

```diff
@@ -31,6 +38,38 @@ const AppContent: React.FC<{ passthrough: unknown }> = observer(({ passthrough }
     const { data } = useRemoteConfig(isMounted());
     const { marketing_growthbook, tracking_datadog, tracking_rudderstack } = data;
 
+    const account_type = LocalStore?.get('active_loginid')
+        ?.match(/[a-zA-Z]+/g)
+        ?.join('');
+
+    const attributes = {
+        account_type: account_type === 'null' ? 'unlogged' : account_type,
+        app_id: String(getAppId()),
+        device_type: store?.ui?.is_mobile ? 'mobile' : 'desktop',
+        device_language: navigator?.language || 'en-EN',
+        user_language: getLanguage().toLowerCase(),
+        country: Cookies.get('clients_country') || Cookies?.getJSON('website_status'),
+    };
+
+    React.useEffect(() => {
+        const fetchIsPasskeySupported = async () => {
+            try {
+                await gb.setAttributes(attributes);
+                await gb.loadFeatures();
+                const is_passkeys_supported = await Promise.all([
+                    browserSupportsWebAuthn(),
+                    gb.isOn('web_passkeys'),
+                    gb.isOn('service_passkeys'),
+                ]);
+                store.client.setIsPasskeySupported(is_passkeys_supported.every(Boolean));
+            } catch (e) {
+                //error handling needed
+            }
+        };
+        fetchIsPasskeySupported();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Can this be avoided by providing the dependency list?

Reply:
well.. actually we only need it while first loading.
but I guess it is safe to add store.client here